tests/cases/data/source/MongoDbTest.php

```diff
@@ -442,122 +458,13 @@ public function testCreate() {
 		$this->assertEqual($expected, $result);
 	}
 
-	public function testCalculation() {
-		$this->_db->connection->results = [new MockResultResource(['data' => [5]])];
-		$this->assertIdentical(5, $this->_db->calculation('count', $this->_query));
-	}
-
 	public function testEnabled() {
 		$this->assertTrue(MongoDb::enabled());
 		$this->assertTrue(MongoDb::enabled('arrays'));
 		$this->assertTrue(MongoDb::enabled('booleans'));
 		$this->assertTrue(MongoDb::enabled('relationships'));
 	}
 
-	public function testArbitraryMethodCalls() {
-		$this->assertInternalType('array', $this->_db->listDBs());
-	}
-
-	public function testDocumentSorting() {
```

Comment:
Seems like an important test.

Reply:
Some tests has been removed because they were more testing mocked code that actually real code. So this one for example has been converted as an integration test in `tests/integration/data/MongoDbTest.php`

Reply:
Okay good that this moved to the right place. You're right, some unit tests - especially the `data` ones - seem like integration tests to me.